samples/ControlCatalog.Web/wwwroot/css/app.css

```diff
@@ -0,0 +1,91 @@
+﻿@import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
```

Comment:
Entire Blazor's default wwwroot shall be purged.

Reply:
purged..

Reply:
some of it.